app/src/main/java/com/example/mycalculator/mvp/CalculatorContract.java

```diff
@@ -0,0 +1,18 @@
+package com.example.mycalculator.mvp;
+
+public interface CalculatorContract {
+
+    public interface calculatorPresenter {
```

Comment:
maybe you can change the name so the contract will be called **CalculatorPresenterContract** and the implementation **CalculatorPresenter**.

Reply:
All the class names should start with upper case

Reply:
Done. Thx!!